rest_sensors.cpp

```diff
@@ -1825,6 +1825,89 @@ int DeRestPluginPrivate::changeSensorConfig(const ApiRequest &req, ApiResponse &
                     }
                 }
             }
+            
+            //Special part for metering interfaces
+            if (sensor->type() == "ZHAConsumption")
+            {
+                if (rid.suffix == RConfigPulseConfiguration)
+                {
+                    if (map[pi.key()].type() == QVariant::Double)
+                    {
+                        if (sensor->modelId() == QLatin1String("ZHEMI101"))
+                        {
+                            quint16 pulseConfiguration = map[pi.key()].toUInt(&ok);
+
+                            if (addTaskSimpleMeteringReadWriteAttribute(task, deCONZ::ZclWriteAttributesId, 0x0300, deCONZ::Zcl16BitUint, pulseConfiguration, VENDOR_DEVELCO))
```

Comment:
Extra points if `ok` is also used :)

```cpp
if (ok && addTask...)
```

Reply:
New highscore then ;)

Even more points for correcting it throughout the file...